mne/viz/evoked.py

```diff
@@ -1498,21 +1651,48 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
         parameters will be passed to the line plot call of the corresponding
         condition, overriding defaults.
         E.g., if evokeds is a dict with the keys "Aud/L", "Aud/R",
-        "Vis/L", "Vis/R", `styles` can be `{"Aud/L":{"linewidth":1}}` to set
+        "Vis/L", "Vis/R", `styles` can be `{"Aud/L": {"linewidth": 1}}` to set
         the linewidth for "Aud/L" to 1. Note that HED ('/'-separated) tags are
         not supported.
+    vlines : "auto" | list of float
+        A list in seconds at which to plot dashed vertical lines.
+        If "auto" and 0. is in the time interval of interest, it is set to [0.]
+        so a vertical bar is plotted at time 0.
```

Comment:
If "auto" and 0. ms is the time point of interest, it is set to [0.] and a vertical bar is plotted at tim